src/main/java/org/folio/marc/migrations/domain/entities/MarcRecord.java

```diff
@@ -0,0 +1,7 @@
+package org.folio.marc.migrations.domain.entities;
+
+import java.util.UUID;
+import org.folio.marc.migrations.domain.entities.types.RecordState;
+
+public record MarcRecord(UUID marcId, UUID authorityId, Object marc, RecordState state, Integer version) {
```

Comment:
Should it be AuthorityMarcRecord or should authorityId be a recordId?

Reply:
Is it entitty for persistence? If not let's move it from "entities" package

Reply:
Changed `authorityId` to `recordId`.
Those are entities retrieved from database view. Or are you asking if we using them with JPA? If so - no